azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -725,6 +743,22 @@ func expandKubernetesClusterAddonProfiles(d *schema.ResourceData) map[string]*co
 		}
 	}
 
+	aciConnector := profile["aci_connector_linux"].([]interface{})
+	if len(aciConnector) > 0 {
+		value := aciConnector[0].(map[string]interface{})
+		config := make(map[string]*string)
+		enabled := value["enabled"].(bool)
+
+		if subnetName, ok := value["subnet_name"]; ok {
+			config["SubnetName"] = utils.String(subnetName.(string))
+		}
+
+		addonProfiles["aciConnectorLinux"] = &containerservice.ManagedClusterAddonProfile{
```

Comment:
just double-checking the casing on these since `aciConnectorLinux` and `SubnetName` are different cases?

Reply:
Good question. Casing does not seem to matter. I changed it to lower case so everything is the same.